Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -489,6 +489,35 @@ def audit_conflicts
     EOS
   end
 
+  def audit_keg_only_style
```

Comment:
Great work on this 👏  Want to try it out with `brew audit --only=keg_only_style` and see how the results look?